statsmodels/graphics/tsaplots.py

```diff
@@ -201,10 +201,7 @@ def seasonal_plot(grouped_x, xticklabels, ylabel=None, ax=None):
     ticks = []
     for season, df in grouped_x:
         df = df.copy() # or sort balks for series. may be better way
-        if pandas.__version__ < '0.17.0':
-            df.sort()
-        else:
-            df.sort_values(inplace=True)
+        df.sort(inplace=True)
```

Comment:
In newer pandas, `sort_values` *requires* a list of keys to sort (at least for data frames). Otherwise, we need to use `sort_index`.  when replacing `sort` with `sort_values` systematically, I had this issue in quite a few places.

But in any case, I am preparing an environment with old versions of numpy, scipy and pandas to check all these and correct them.